kirin/gtfs_rt/tasks.py

```diff
@@ -114,15 +84,55 @@ def _is_last_call_too_recent(func_name, contributor, minimal_call_interval):
     return False
 
 
+def _update_etag(resp, config):
```

Comment:
```suggestion
def _update_redis_etag(resp, config):
```

Also, can you protect it to just log if this raises, please? This ETag management is a "bonus", and we want this to work even if redis is KO.